stdlib/public/Concurrency/YieldingContinuation.swift

```diff
@@ -95,6 +95,7 @@ extension YieldingContinuation where Failure == Error {
   ///
   /// When multiple calls are awaiting a produced value from next any call to
   /// yield will resume all awaiting calls to next with that value. 
+  @available(*, deprecated, message: "`YieldingContinuation` was replaced by `Series` and will be removed shortly.")
```

Comment:
These messages are out of date

Reply:
this will be resolved by the other incoming PR that removes it wholesale - this file isn't even being compiled anymore w/ this PR, it was removed from the CMakeLists.txt